### PR TITLE
Update modules

### PR DESCRIPTION
**Pull request by ramprasadn**
## PR checklist

This PR updates two modules (eklipse & svdb query) and introduces along relevant changes in the pipeline.

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/raredisease/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/raredisease _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test_one_sample,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: subworkflows/local/annotate_structural_variants.nf [hunk 1]
```diff
@@ -33,16 +37,62 @@ workflow ANNOTATE_STRUCTURAL_VARIANTS {
...
+            []
         )
 
-        PICARD_SORTVCF(SVDB_QUERY.out.vcf, ch_genome_fasta, ch_genome_dictionary)
+        ch_vcf
+            .join(SVDB_QUERY_DB.out.vcf, remainder: true)
+            .branch { it  ->
+                call: it[2].equals(null)
+                    return [it[0], it[1]]
+                db: !(it[2].equals(null))
```
Comment (jemten) [inline subworkflows/local/annotate_structural_variants.nf:66]: Is it possible to use some more descriptive branch names? I guess that the call branch is basically the original vcf channel and the db branch is the possible output from the SVDB_QUERY:DB output in the case where it was part of the run. Correct?

File: subworkflows/local/annotate_structural_variants.nf [hunk 2]
```diff
@@ -33,16 +37,62 @@ workflow ANNOTATE_STRUCTURAL_VARIANTS {
...
+            [],
+            ch_svdb_bedpedbs.bedpedbs.toList()
+        )
+
+        ch_querydb_out
+            .join(SVDB_QUERY_BEDPE.out.vcf, remainder: true)
+            .branch { it  ->
+                db: it[2].equals(null)
+                    return [it[0], it[1]]
+                bedped: !(it[2].equals(null))
```
Comment (jemten) [inline subworkflows/local/annotate_structural_variants.nf:88]: same comment about branch name

**Review by jemten — COMMENTED** [review]
Looks good, just a comment about the names of the channel branches. It took me a couple of minutes to figure out which branch was what. Or maybe that is as easy as it will get. Also, the svdb_query prefix which should not cause any issues for now but is a potential headache.

File: modules/nf-core/svdb/query/main.nf [hunk 3]
```diff
@@ -13,7 +13,8 @@ process SVDB_QUERY {
     val(in_frqs)
     val(out_occs)
     val(out_frqs)
-    path (vcf_dbs)
+    path(vcf_dbs)
+    path(bedpe_dbs)
 
     output:
     tuple val(meta), path("*_query.vcf")    , emit: vcf
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply modules/nf-core/svdb/query/main.nf:20]: SVDB's output carries that suffix by default https://github.com/J35P312/SVDB/blob/c8c538089fd5f05d891db3686d83a79c97d5adbf/svdb/__main__.py#L32

**Review by Lucpen — COMMENTED** [review]
Looks good, I will need to look into the new module tests

File: modules/nf-core/eklipse/main.nf [hunk 4]
```diff
@@ -4,7 +4,7 @@ process EKLIPSE {
     label 'process_single'
 
     // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.
-    conda "bioconda::eklipse=1.8"
+    conda "${moduleDir}/environment.yml"
```
[the remark this answers is not in the document]
Comment (ramprasadn) [reply modules/nf-core/eklipse/main.nf:7]: Yupe.. with nf-tests every tool and subtool will come as a unit containing scripts, tests, and the environment  :)

**Review by jemten — APPROVED** [review]
Looks good! Could you just check that SVDB_query works as intended when you run both a vcf and bedpe database. Right now they will have the same suffix and I don't know which file be selected to the emit channel. Go ahead and merge when you feel confident in the code

**Pull request merged**